webhooks-extension/README.md

```diff
@@ -11,20 +11,26 @@ See our [Getting Started](https://github.com/tektoncd/experimental/blob/master/w
 ### Install Prereqs
 
 [Install and Configure Prereqs](./docs/InstallPrereqs.md)  
-[Additional Configuration for installing and using Red Hat Openshift](./docs/InstallOnOpenshift.md)  
 
 ### Install Webhook Extension
 
-[Installing Official Release (stable)](./docs/InstallReleaseBuild.md)  
+To install an official release please navigate to the docs on one of those branches. The link below does not contain all the information needed at this time.
```

Comment:
nit: This sentence is unclear to me, it took me a few times to figure out what `those branches` referred to.

Can you please revise it to something like `...on one of the release branches.`

Reply:
I've changed it to....

```
To install an official release please navigate to the docs for that release. In the branches dropdown at the top of this page, simply select the branch name matching the version you want to install.
The link below does not contain all the information needed at this time - this message can be removed after the 0.3 release occurs. 
```

Good enough? It's only temporary .... until we have a 0.3 release and I have added extra words on the InstallReleaseBuild.md now to also note different prereqs and actions for installing 0.2.1 and earlier (which will need to remain even after the 0.3 release)

Reply:
Yep, thanks Duane!